wordpress-mobile/WordPress-iOS
# Issue/4707 widget layout

Fixes #4707 

Warning: Stackviews on stackviews on stackviews

To test:

See both the widget configured and not configured modes. Also, verify that the widget scales correctly when changing the system text size (Settings > Display & Brightness > Text Size)

![img_0605](https://cloud.githubusercontent.com/assets/1158819/19580084/e7557cd0-96d8-11e6-8a8f-fb1514d4bb56.PNG)
![img_0606](https://cloud.githubusercontent.com/assets/1158819/19580083/e7207abc-96d8-11e6-98c2-7beb5b289153.PNG)

Needs review: @astralbodies
